src/librbd/journal/Types.h

```diff
@@ -121,12 +121,13 @@ struct OpFinishEvent : public OpEventBase {
 
 struct SnapEventBase : public OpEventBase {
   std::string snap_name;
+  cls::rbd::SnapshotNamespace snap_namespace;
```

Comment:
Per my conversation with Xing: http://git.net/ml/openstack-dev/2016-12/msg00323.html
When we clone a consistency group we basically create a new image from every snapshot in this group snapshot. Then we make a group of those images.